test/e2e/e2e_suite_test.go

```diff
@@ -0,0 +1,63 @@
+package e2e_test
+
+import (
+	"fmt"
+	"os"
+	"testing"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+const (
+	EventuallyTimeout   = 60 * time.Second
+	ConsistentlyTimeout = 1 * time.Second
+	// TODO data provider?
+	ConfigAll     = "../../deploy/all-in-one.yaml" // basic configuration (release)
+	ProjectSample = "data/atlasproject.yaml"
+	ClusterSample = "data/atlascluster_basic.yaml"
+)
+
+var (
+	// default
+	Platform   = "kind"
+	K8sVersion = "v1.17.17"
+)
+
+func TestE2e(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "E2e Suite")
+}
+
+var _ = SynchronizedBeforeSuite(func() []byte {
```

Comment:
[q] why do we need the `SynchronizedBeforeSuite` instead of the `BeforeSuite`?

Reply:
this check run once instead of each time